WcaOnRails/app/models/user.rb

```diff
@@ -406,6 +406,10 @@ def can_edit_delegate_report?(delegate_report)
     can_admin_results? || (competition.delegates.include?(self) && !delegate_report.posted?)
   end
 
+  def can_see_admin_competitions?
+    board_member? || senior_delegate?
+  end
```

Comment:
Any update here? As discussed offline, I would like for us to stop using the word "admin", as it is used in so many different ways, I fear it has no meaning anymore. Is there a better word for this page?

Going back to @jonatanklosko's point, we definitely want to give admins access to this page. Maybe the results team won't find it useful, but if there's ever a bug on the page, we want the software team to have access to it to debug.